examples/finance/bs_ivbp.ipynb

```diff
@@ -37,13 +37,12 @@
    ],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    np.mean(vals)
</pre><p>This should be a norm check instead + update the text below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>